arch/Kconfig

```diff
@@ -516,7 +523,7 @@ config MPU_GAP_FILLING
 menuconfig FLOAT
 	bool "Floating point"
 	depends on CPU_HAS_FPU
-	depends on ARM || X86 || ARC
+	depends on ARC || ARM || RISCV || X86
```

Comment:
passing comment: really should just delete this line and rely solely on CPU_HAS_FPU which should be selected by CPUs that have this capability